public/docs/ts/latest/guide/index.jade

```diff
@@ -22,7 +21,7 @@ table(width="100%")
   col(width="15%")
   col
   tr(style=top)
-    td <b>QuickStart</b>
+    td <b><a href="../quickstart">QuickStart</a></b>
```

Comment:
It's a little weird to have links on QuickStart but not on Guide & Advanced... But it *is* definitely handy to have these links, especially for those times when the window is too narrow to display the sidenav.

This is a case where having everything in the guide/ directory is coming back to bite us.

Maybe we should try this out and see if anyone objects?